internal/ui/ui_glfw.go

```diff
@@ -860,10 +1006,17 @@ func (u *userInterfaceImpl) initOnMainThread(options *RunOptions) error {
 		glfw.WindowHint(glfw.Visible, glfw.True)
 	}
 
+	// Get our target monitor.
+	monitor := u.getInitWindowMonitor()
+
+	if monitor != glfw.DontCare {
+		setInitMonitor(monitors[monitor].m)
+	}
```

Comment:
In most cases, is `setInitMonitor` is called twice? (one is at `initialize`) I feel like this is redundant. Can we integrate them?

Reply:
Please see latest commit. This seems to work fine. `setInitMonitor` could be rolled into `initOnMainThread` since it is only ever called there.

I am not 100% this was the correct way to go about it. The important bit that I removed was:
```
	if max := u.initFullscreenWidthInDIP - ww; wx >= max {
		wx = max
	}
	if max := u.initFullscreenHeightInDIP - wh; wy >= max {
		wy = max
	}
	u.setWindowPositionInDIP(wx, wy, u.initMonitor)
```
If this was not removed, then when `ebiten.SetMonitor` was not called, the window would end up in the top-left.

Reply:
Thanks.

> If this was not removed, then when ebiten.SetMonitor was not called, the window would end up in the top-left.

I realized that `initFullscreenWidthInDIP` affected the result of `ui.ScreenSizeInFullscreen`. This means that the result of `ebiten.ScreenSizeInFullscreen` BEFORE `RunGame` would change.

I'm really sorry but could you revert the change and call `setInitMonitor` at `initialize()e` again?

Reply:
Done